packages/webapp/src/components/CropCatalogue/CropStatusInfoBox/index.jsx

```diff
@@ -1,60 +1,45 @@
-import { makeStyles } from '@mui/styles';
+/*
+ *  Copyright 2024 LiteFarm.org
```

Comment:
I appreciate you how went above and beyond, but these files are not from 2024 😂 You can see in the git history. So I guess if you were going to do this it would be 2021-2024?

Reply:
C'mooooon hahaha... 😂  They have been licensed since 2024 which is an accurate statement. We should be adding to old files that we touch technically.

And tbh I still believe we had a conversation with Ivan where just the most recent year is needed -- see main copyright notice on other websites.
![Screenshot 2024-09-13 at 8 00 30 AM](https://github.com/user-attachments/assets/ebcbb697-a53c-49be-9876-000f10118107)

It would be more maintainable to update all every year than try to keep track of when it was first made.

Reply:
The conversation we had with Ivan was about how we used to copy-paste a license format like this:

```js
// *  Copyright 2019, 2020, 2021, 2022, 2023 LiteFarm.org
 ```
 
Even for _new_ files, and he corrected us that it should only track the history of that particular file, not dates of the whole project. And [this](https://www.gnu.org/licenses/gpl-howto.en.html#copyright-notice) is the guideline about GNU copyright that he shared with us in tech daily with the date range format described.
 
> It would be more maintainable to update all every year than try to keep track of when it was first made.

Agreed it would be easier, but we already have a mess of license patterns in the codebase, why are we establishing another one that says copyright refers to time created OR time last touched?

(And why are you c'moooooning it?? You didn't have to add to all files, just the new file, per the checklist!)

Reply:
Lol I made the checklist! .... I think we should add to all files touched! 

I was just cm'oning the bane of my existence the copyright notice haha.. its the most useless thing I forget every time and even when I remember it ... its wrong haha! 

I should really make a formal request to allow the LICENCE in the top of repo be all we need -- it seems like its not expected to do it in every file anymore..

Reply:
range added!

Reply:
I realized reading the document above (and our Confluence guidelines) that apparently the range method is no good unless it was touched literally every year in that range. Sorry 😂 Also the other touched file says 2024 still. But even I am getting the c'mon feeling now so let's get these two lines of CSS merged 🤣 

I made a [ticket](https://lite-farm.atlassian.net/browse/LF-4441) for copyright discussion.